apis/v1alpha1/opentelemetrycollector_webhook_test.go

```diff
@@ -207,9 +207,9 @@ func TestOTELColValidatingWebhook(t *testing.T) {
     endpoint: "0.0.0.0:12346"
   prometheus:
     config:
-      scrape_config:
-        job_name: otel-collector
-        scrape_interval: 10s
+      scrape_configs:
```

Comment:
did this not work previously?

Reply:
It was used in admission tests that (until now) didn't care about the actual contents of the Prometheus receiver config, so it did work.